src/main/java/seedu/address/logic/parser/FindCommandParser.java

```diff
@@ -118,10 +118,11 @@ public FindCommand parse(String args) throws ParseException {
         if (argMultimap.getValue(PREFIX_NATIONALITY).isPresent()
             && !argMultimap.getValue(PREFIX_NATIONALITY).get().trim().isEmpty()) {
             List<String> nationalityKeywords = argMultimap.getAllValues(PREFIX_NATIONALITY);
-
+            /*
             for (String nationality : nationalityKeywords) {
-                ParserUtil.parseName(nationality);
+                ParserUtil.parseNationality(nationality);
             }
+             */
```

Comment:
Is this commented code still needed? If not I think would be better to remove it

Reply:
Don't remove it. I'll see if i can resolve it later. Same for the one below